source/server/ScriptEngine.h

```diff
@@ -40,237 +40,241 @@ class Sequencer;
-
-    int streamAdded(int uid, RoRnet::StreamRegister *reg);
-
-    int playerChat(int uid, std::string msg);
-
-    void gameCmd(int uid, const std::string &cmd);
-
-    int framestep(float dt);
-
-    /**
-     * A loop that makes sure that does a regular call to the frameStep method.
-     * @see frameStep
-     */
-    void timerLoop();
-
-    /**
-     * Gets the currently used AngelScript script engine.
-     * @return a pointer to the currently used AngelScript script engine
-     */
-    asIScriptEngine *getEngine() { return engine; };
-
-    /**
-     * Sets an exception that aborts the currently running script and shows the exception in the log file.
-     * @param message A descriptive error message.
-     */
-    void setException(const std::string &message);
-
-    /**
-     * Adds a script callback.
-     * @param type The type of the callback. This can be one of the following: 'frameStep', 'playerChat', 'gameCmd', 'playerAdded', 'playerDeleted'.
-     * @param func A pointer to a script function.
-     * @param obj A pointer to the object of the method or NULL if func is a global function.
-     */
-    void addCallback(const std::string &type, asIScriptFunction *func, asIScriptObject *obj);
-
-    /**
-     * This method checks and converts the parameters and then adds a script callback.
-     * @param type The type of the callback. \see addCallback
-     * @param func The name of a script function.
-     * @param obj A pointer to the object of the method or NULL if func is a global function.
-     */
-    void addCallbackScript(const std::string &type, const std::string &func, asIScriptObject *obj);
-
-    /**
-     * Deletes a script callback.
-     * @param type The type of the callback. \see addCallback
-     * @param func A pointer to a script function.
-     * @param obj A pointer to the object of the method or NULL if func is a global function.
-     */
-    void deleteCallback(const std::string &type, asIScriptFunction *func, asIScriptObject *obj);
-
-    /**
-     * This method checks and converts the parameters and then deletes a script callback.
-     * @param type The type of the callback. \see addCallback
-     * @param func The name of a script function.
-     * @param obj A pointer to the object of the method or NULL if func is a global function.
-     */
-    void deleteCallbackScript(const std::string &type, const std::string &_func, asIScriptObject *obj);
-
-    /**
-     * Deletes all script callbacks.
-     */
-    void deleteAllCallbacks();
-
-    /**
-     * This checks if a script callback exists.
-     * @param type The type of the callback. \see addCallback
-     * @param func A pointer to a script function.
-     * @param obj A pointer to the object of the method or NULL if func is a global function.
-     * @return true if the callback exists
-     */
-    bool callbackExists(const std::string &type, asIScriptFunction *func, asIScriptObject *obj);
+ScriptEngine(Sequencer *seq);

```

Comment:
Class members absolutely must be indented (I don't even remember seeing it unindented anywhere). It should also be consistent with `struct` which is indented here.